apis/src/api/v1/auth/get_identity_handler.rs

```diff
@@ -0,0 +1,14 @@
+use super::auth::Auth;
+use actix_web::get;
+use actix_web::HttpResponse;
+use serde_json::json;
+
+#[get("/api/v1/auth/id")]
```

Comment:
Why do we need that endpoint ?

Reply:
This is basically a "health check" for you token. Helped me debug things, might help others. We might also use this to return some meaningful things about your account, like elo, games played, ...